src/index.js

```diff
@@ -79,11 +79,22 @@ async function eventHandle(track, key, e) {
 }
 
 // Export out init() function that kicks everything off
-export default function init(key) {
+export default function init(key, config) {
   if (!key) {
     throw new Error('A segment key must be passed to init');
   }
 
+  // If analytics are explicitly disabled
+  if (config && config.disableBrowserAnalytics) {
+    return {
+      track: () => {},
+      identify: () => {},
+      page: () => {},
+      group: () => {},
+      alias: () => {},
+    };
+  }
```

Comment:
I think it would be better if we just wrap a check around [this line](https://github.com/festicket/analytics/blob/master/src/index.js#L90)?

```js
if(config.trackClicks) {
  window.addEventListener('click', e => eventHandle(track, key, e));
}
```

If we do it this way (explicit opt out) then the applications implementation is simpler?:

```js
// in some project somewhere:
export { track, identify, page, group, alias } = init(KEY, { 
  trackClicks: process.browser 
})
```